server.py

```diff
@@ -83,15 +83,31 @@ def page_action():
 	suit = bottle.request.forms.get('suit')
 	# Make sure that only actual players perform actions.
 	if 0 <= player < 4:
-		# Play a card
+		# Play or discard a card
 		if action == 'play':
 			if card in game_data['hands'][player]:
-				game_data['table'].append({
-					'player': player,
-					'card': card,
-					'state': 'normal'
+				if len(game_data['hands'][player]) <= 10:
+					# in this case, play a card
+					# make sure that only one card is played per turn
+					already_played = 0

```

Comment:
`True` and `False` is preferred.

(I know I said this elsewhere, I just wanted an excuse to try out this feature :smile:)